### ISSUE TITLE
git executable required

### ISSUE THREAD

**Issue by pgajdos** [doxygen/doxygen#7285]
Thanks for 1.8.16 release.

There, git executable is required, not optional as declared. I do not feel to know cmake to the extent to provide a pull request, but removing word `REQUIRED` from here:
```
if(NOT DEFINED GIT_EXECUTABLE)
    find_package(Git QUIET REQUIRED)
 endif()
```
fixed the issue for me.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7291

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.17.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by pgajdos** [comment]
Thanks!
